README.md

```diff
@@ -13,6 +13,7 @@ Single-server Discord bot that makes counting competitive.
 ## Deployment
 
 1. Generate an OAuth invite link on the [Discord Developer Portal](https://discord.com/developers/applications/).
+    * In the OAuth tab, select the `bot` scope and these permissions: `send messages, embed links, attach files, read message history, add reactions`.
```

Comment:
nitpick but maybe add the number it generates as well in the case someone just needs that?